docs/source/deployment/gcp-functions.mdx

```diff
@@ -54,7 +49,11 @@ Click **Create Function**. Give the function a name and set the **Trigger type**
 
 For quick setup and access to the GraphQL endpoint/landing page, choose to **Allow unauthenticated invocations**. To require authentication for this endpoint, you can manage authorized users via [Cloud IAM](https://console.cloud.google.com/iam-admin/iam).
 
-Save your configuration changes in the Trigger section. Copy the trigger's URL for later and click **Next**.
+Save your configuration changes in the Trigger section. Copy the trigger's URL for later.
+
+Expand the "Runtime, Build, and Connections Settings" panel and add a runtime environment variable named `NODE_ENV` with value `development`. (This overrides Cloud Function's default of `production` and will allow you to easily test your graph with Apollo Sandbox. You can delete this environment variable once you're ready for your app to be considered a production app.)
```

Comment:
If you can think of a way that users can be reminded this is set that could be helpful? I could easily see this being forgotten, esp. by a GCP/cloud beginner. If not I don't think it's a huge deal.

Reply:
Yeah maybe add a section at the end.